backend/src/plugins/fastify-roles/index.js

```diff
@@ -29,12 +29,11 @@ async function fastifyRoles(fastify, opts) {
    *
    * @return  {[type]}           [return description]
    */
-  function onRequest(request, reply, done) {
```

Comment:
mmm... are you sure this still works, with out the done, I thought this had to be there....